gen/definitions/classes.yaml

```diff
@@ -164,3 +164,33 @@ l3extProvLbl:
   ui_locations:
     - "Tenants (infra) -> Networking -> L3Outs -> Policy -> Main -> Provider Label"
   sub_category: "L3Out"
+
+netflowMonitorPol:
+  resource_name: "netflow_monitor_policy"
+  ui_locations:
+    - "Tenants -> Policies -> NetFlow -> NetFlow Monitors"
+  sub_category: "Tenant Policies"
+
+netflowRsMonitorToRecord:
+  resource_name: "relation_to_netflow_record"
+  ui_locations:
+    - "Tenants -> Policies -> NetFlow -> NetFlow Monitors"
+  sub_category: "Tenant Policies"
+
+netflowRsMonitorToExporter:
+  resource_name: "relation_to_netflow_exporter"
+  ui_locations:
+    - "Tenants -> Policies -> NetFlow -> NetFlow Monitors"
+  sub_category: "Tenant Policies"
+
+netflowExporterPol:
+  resource_name: "netflow_exporter_policy"
+  ui_locations:
+    - "Tenants -> Policies -> NetFlow -> NetFlow Exporters"
+  sub_category: "Tenant Policies"
+
+netflowRecordPol:
+  resource_name: "netflow_record_policy"
+  ui_locations:
+    - "Tenants -> Policies -> NetFlow -> NetFlow Records"
+  sub_category: "Tenant Policies"
```

Comment:
add ui locations for the non tenant DNs here?

Reply:
will do, thanks!